### DISCUSSION TITLE
Installer/Uninstaller/Updater

### DISCUSSION THREAD

**Discussion by gjf** [valinet/ExplorerPatcher#287] [General]
Hi.

Firstly - thank you for this tool, I really liked it once I got update to Win11.

The only thing that makes me crazy a little - a necessity to manually update process. I am lazy to perform all manual rename/copy every time - so I hyave created a manual installer/uninstaller/updater.

It is simple as crap and available here: https://github.com/gjf/Windows-11-ExplorerPatcher-Installer

You can include it with releases or simply copy the logic into your project - it's up to you. But I really think you have a lot of ideas to implement and maintain - so you can simply use what I have created.

And again: THANK YOU!

**Comment by valinet** [comment]
Thanks, I am planning to maybe do an installer/updater at some point.

**Comment by rishavs** [comment]
Please do consider adding support for app stores like chocolatey, scoop, winget etc.
They are an immense QoL improvement.

**Reply by valinet** [reply]
Who is going to maintain those?

**Reply by BraINstinct0** [reply]
umm, there's a simple solution.
Allow, but make it mandatory to note as 'unofficial'.
That way ones who are willing to, and have the skills and time will do the job.
Or maybe not, but that doesn't make any problem ;)

**Comment by AceWurld** [comment]
please how can i uninstall this explorer patcher cause it's affecting my side icons like battery and wifi on windows 11.

**Comment by gailium119** [comment]
I think the installer should add a command-line option that enables it to install in an offline image like ep_install.exe /offline /image:C:\mount

**Reply by valinet** [reply]
Just drop `dxgi.dll` in `C:\Windows` and `C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy`. You'll only miss the uninstall data from the registry, so it won't show an entry in the Control Panel. But you can merge that info manually in WDS (I think) when customizing the image or it will be created right away at first boot when you will probably see an update, update and that creates the registry info. Supporting this correctly in the code is too... complicated (have to load the registry hive from that image, have to determine the Windows directory correctly somehow etc). I have to take care of enough things already... I am sure you can also hack it away using some PowerShell script...

**Comment by Sqwrrly** [comment]
Just chiming in that I would love chocolatey support.

**Comment by ryzendew** [comment]
need to uninstall please
